enhancements/oc/must-gather.md

```diff
@@ -64,16 +64,20 @@ ship a second one.
 
 `must-gather` for openshift is a combination of three tools:
 
-1. A client-side `inspect` command that works like a super-get.  It has semantic understand of some resources and traverses
-   links to get interesting information beyond the current.  Pods in namespaces and logs for those pods as a for instance.
-   Currently this is `openshift-must-gather inspect`, but we are porting this to `oc adm` as experiemental in 4.3.  We may
+1. A client-side `inspect` command that works like a super-get. It has semantic understanding of some resources and traverses
+   links to get interesting information beyond the current. Pods in namespaces and logs for those pods, for instance.
+   Currently, this is `openshift-must-gather inspect`, but we are porting this to `oc adm` as experimental in 4.3. We may
    change and extend arguments over time, but the intent of the command will remain.
```

Comment:
Should we add that collections scripts should leverage this when possible; but it is optional?

Reply:
I don't know the must-gather internals enough to comment on this. :(

Reply:
@ardaguclu or @ingvagabund can you take a followup item to update the above. It is quite outdated and would be good to also add what Eric points out above.